ui/src/app/workflows/components/workflow-filters/workflow-filters.tsx

```diff
@@ -11,7 +11,10 @@ import {NamespaceFilter} from '../../../shared/components/namespace-filter';
 import {TagsInput} from '../../../shared/components/tags-input/tags-input';
 import {services} from '../../../shared/services';
 
+import {InputFilter} from '../../../shared/components/input-filter';
 import './workflow-filters.scss';
+import {DropDown} from '../../../shared/components/dropdown/dropdown';
+import classNames from 'classnames';
```

Comment:
please follow the existing import order conventions: external imports at the top, then external CSS, then internal imports, then internal CSS (also yes an ESLint or `prettier` plugin would be great)

Reply:
Yes we could work on that, I was assuming that Eslint was already reordering for me 🙃

Reply:
I had to manually fix this lots of these issues in https://github.com/argoproj/argo-workflows/pull/13813 as they are partly unsortable by tooling since they were placed after a side effect import